src/annotator/components/adder-toolbar.js

```diff
@@ -17,7 +17,9 @@ function ToolbarButton({ badgeCount, icon, label, onClick, shortcut }) {
       aria-label={title}
       title={title}
     >
-      {icon && <SvgIcon name={icon} />}
+      {icon && (
+        <SvgIcon name={icon} className="annotator-adder-actions__icon" />
```

Comment:
By having a `className` that applies to the icon specifically in this button and not the text label, they can be styled independently of each other (allowing a color for the icon as `$grey-mid` but the text as `$color-text`).